scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestController_SpecificSchedule_Test.java

```diff
@@ -77,23 +80,28 @@ public void beforeTest() throws SchedulerException {
 	@After
 	@Transactional
 	public void afterTest() {
-		for (ScheduleEntity entity : scheduleDao.findAllSchedulesByAppId(appId)) {
-			scheduleDao.delete(entity);
+		for (String appId : multiAppIds) {
+			for (ScheduleEntity entity : scheduleDao.findAllSchedulesByAppId(appId)) {
+				scheduleDao.delete(entity);
+			}
 		}
 	}
 
 	@Test
 	@Transactional
 	public void testGetAllSchedule_with_no_schedules() throws Exception {
-		ResultActions resultActions = callGetAllSchedulesByAppId();
+		ResultActions resultActions = callGetAllSchedulesByAppId(appId);
 		assertSpecificSchedulesFoundEquals(0, resultActions);
 	}
 
 	@Test
 	@Transactional
 	public void testCreateAndGetSchedules() throws Exception {
-		assertCreateAndGetSchedules(1);
-		assertCreateAndGetSchedules(5);
+		assertCreateAndGetSchedules(singleAppId, 1);
+		assertCreateAndGetSchedules(singleAppId, 5);
```

Comment:
You are missing the point. If you have both singleAppId and multiAppIds tests, what specifically is the singleAppId testing that isn't covered by multiAppIds.
If singleAppId testing fails, wouldn't multiAppId testing also fail? The reverse is not necessarily true though.